SDWebImage/SDWebImageDownloader.h

```diff
@@ -85,9 +85,6 @@ typedef NS_OPTIONS(NSUInteger, SDWebImageDownloaderOptions) {
     SDWebImageDownloaderPreloadAllFrames = 1 << 11
 };
 
-FOUNDATION_EXPORT NSString * _Nonnull const SDWebImageDownloadStartNotification;
-FOUNDATION_EXPORT NSString * _Nonnull const SDWebImageDownloadStopNotification;
```

Comment:
This is `extern`, can use any times, if we wants to remove, remove extern in download file. Not import any break, because download header file include downloader header.

Reply:
SDWebImage 5.0, it’s designed to import the umbrella headers file <SDWebImage/SDWebImage.h>`. The standard way for framework.

Import subtree of files, it’s not considered as API break.

For exmaple, if I move one typedef, from `SDWebImageManager.h`, into `SDWebImageDownloader.h`. It’s allowed for minor version upgrade.

This is also what you use <UIKit/UIKit.h>.

But anyway, for this one, I can just move all of them into downloader.